core-bundle/contao/themes/flexible/styles/main.css

```
@@ -169,7 +169,7 @@ body.popup {
 	background: var(--content-bg);
 	border: 1px solid var(--content-border);
 	box-shadow: 0 1px 6px rgba(0,0,0,.2);
-	z-index: 2;
+	z-index: 4;
```

```suggestion
	z-index: 4; /* above .jump-targets */
```

Just a nitpick: I think it’s good practise to document all `z-index` values greater than `1`.

---

Taking https://github.com/contao/contao/pull/6747#issuecomment-1900771908 into consideration, I checked the other z-index as well and increased the one for tl_formbody_submit.

The comments were added @ausi